src/js/main.js

```diff
@@ -31,4 +32,203 @@ Site = {
   },
 };
 
-Site.init();
\ No newline at end of file
+Site.StreamChecker = {
+  statusUrl: 'https://public.radio.co/stations/s0b5e9c02c/status',
+  statusCheckInterval: 5000,
+  streamData: null,
+  init: function() {
+    var _this = this;
+
+    _this.startChecker();
+  },
+
+  startChecker: function() {
+    var _this = this;
+
+
+    _this.checkStreamStatus();
+
+    // Start checker on an interval
+    setInterval(function() {
+      _this.checkStreamStatus();
+    }, _this.statusCheckInterval);
+  },
+
+  // Check the stream status thru ajax and change the player accordingly
+  checkStreamStatus: function() {
+    var _this = this;
+
+    // Make the ajax request
+    $.getJSON(_this.statusUrl, function(data) {
+
+      // Save data in a cache var, just in case
+      _this.streamData = data.status;
+
+      // Check if the stream is offline
+      if(data.status == 'offline') {
```

Comment:
nice formatting bro (but really are you relying on both if's being false for error handling?)